checkov/terraform/checks/resource/aws/ECSClusterLoggingEncryptedWithCMK.py

```diff
@@ -0,0 +1,30 @@
+from checkov.common.models.enums import CheckResult, CheckCategories
+from checkov.terraform.checks.resource.base_resource_check import BaseResourceCheck
+
+
+class ECSClusterLoggingEncryptedWithCMK(BaseResourceCheck):
+    def __init__(self):
+        name = "Ensure Cluster logging with CMK"
+        id = "CKV_AWS_224"
+        supported_resources = ['aws_ecs_cluster']
+        categories = [CheckCategories.ENCRYPTION]
+        super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
+
+    def scan_resource_conf(self, conf):
+        if conf.get("configuration"):
+            if conf.get("configuration")[0].get('execute_command_configuration'):
+                command_conf = conf.get("configuration")[0].get('execute_command_configuration')[0]
+                if not command_conf.get('logging') == ['NONE']:
+                    if command_conf.get('kms_key_id'):
+                        log_conf = command_conf.get('log_configuration')[0]
```

Comment:
I think it is possible to not set the `log_configuration` block at all, then this would break. please add an extra if condition or add to the previous one to check for the existence of `log_configuration` -> 
```python
if command_conf.get('log_configuration'):
```

Reply:
You're not wrong